source/upgrading/upgrade/_sysvm_restart.rst

```diff
@@ -52,9 +52,18 @@ in the system VMs.
    With respect to VRs, a network restart without cleanup is initiated to during live patching to ensure all rules
    are re-applied. 
 
-   **NOTE:** In case there is an absolute need to upgrade the system VM template due to availability of
-   security patches or update in a package provided by the template, then the old workflow of recreating the system
-   VM will need to be followed, which would mean noticible downtime.
+   **NOTE:** 
+
+   1. In case there is an absolute need to upgrade the system VM template due to availability of security patches, 
+   update in a package provided by the template, or in case live-patch doesn't seem to work for any reason
```

Comment:
statement not super clear? Can this be written in straight forward instructions way?